### PR TITLE
Magento_CurrencySymbol module. Replace replace deprecated message functions

### PR DESCRIPTION
**Pull request by atwixfirster**
### Description (*)

PR replaces deprecated message functions in the `Magento_CurrencySymbol` module.

### Fixed Issues (if relevant)

1. magento/magento2#<issue_number>: Issue title

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/Customer/Ui/Component/Listing/Address/Column/Actions.php [hunk 1]
```diff
@@ -6,6 +6,9 @@
  */
 namespace Magento\Customer\Ui\Component\Listing\Address\Column;
 
+use Magento\Customer\Api\AddressRepositoryInterface;
```
Comment (VladimirZaets) [inline app/code/Magento/Customer/Ui/Component/Listing/Address/Column/Actions.php]: Hi @atwixfirster. Thanks for collaboration. Looks like this code not related to this PR.

Comment (atwixfirster) [reply app/code/Magento/Customer/Ui/Component/Listing/Address/Column/Actions.php]: Very good point, @VladimirZaets 

Yes, I 've mistakenly committed wrong files. 

Thank you!

**Review by sidolov — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sidolov, thank you for the review.
[ENGCOM-5967](https://jira.corp.magento.com/browse/ENGCOM-5967) has been created to process this Pull Request

**Comment by engcom-Delta** [comment]
:heavy_check_mark: QA passed

**Pull request merged**